PharoProjects/RichText.md

```diff
@@ -60,6 +57,29 @@ textAction := TextAction new
 	actOnClickBlock: [ 
 		Transcript nextPutAll: 'action clicked';cr;endEntry
 		].
+'My action text' asText addAttribute: textAction from: 4 to: 10.
+```
+
+## Inline images
+It is possible to insert images into text. It is done using a `TextAnchor` attribute which can only be put on the text obtained using the expression `(String value: 1) asText`. The following example illustrates this:
+
+```Smalltalk
+anchoredImage := 
+	(String value: 1) asText addAttribute: 
+		(TextAnchor new anchoredMorph: PolymorphSystemSettings pharoLogoForm ).
+
+('Line before image\' withCRs asText), 
+anchoredImage,  
+('\Line afterimage' withCRs asText).
+```
+
+There are further examples in the class comment of `TextAnchor`.
+
+#### Notice.
+The inline images relies on the morphic system. Not all morphs seem to be able to be inlined in text in Pharo.
+
+In addition, you will most likely get the best results if you put images larger than the font size on a line by themself. The class comment of `TextAnchor` has an example of inlining a small icon.
+=======
```

Comment:
```suggestion
```

Reply:
I fucked up the conflict resolution, sorry. :-(

Reply:
Booooouh! ;)